docs/locations.md

```diff
@@ -0,0 +1,165 @@
+
+- If you're using Ark's restic integration, restic data is now stored under a prefix/subdirectory of the main Ark bucket, and will go into the bucket corresponding to the `BackupStorageLocation` selected by the user at backup creation time.
+
+## Examples
+
+Let's look at some examples of how we can use this new mechanism to address each of our previously unsupported use cases:
+
+#### Take snapshots of more than one kind of persistent volume in a single Ark backup (e.g. in a cluster with both EBS volumes and Portworx volumes)
+
+During server configuration:
+
+```shell
+ark snapshot-location create ebs-us-east-1 \
+    --provider aws \
+    --config region=us-east-1
+
+ark snapshot-location create portworx-cloud \
+    --provider portworx \
+    --config type=cloud
+```
+
+During backup creation:
+
+```shell
+ark backup create full-cluster-backup \
+    --volume-snapshot-locations ebs-us-east-1,portworx-cloud
+```
+
+Alternately, since in this example there's only one possible volume snapshot location configured for each of our two providers (`ebs-us-east-1` for `aws`, and `portworx-cloud` for `portworx`), Ark doesn't require them to be explicitly specified when creating the backup:
+
+```shell
+ark backup create full-cluster-backup
+```
+
+#### Have some Ark backups go to a bucket in an eastern USA region, and others go to a bucket in a western USA region
+
+During server configuration:
+
+```shell
+ark backup-location create default \
+    --provider aws \
+    --bucket ark-backups \
+    --config region=us-east-1
+
+ark backup-location create s3-alt-region \
+    --provider aws \
+    --bucket ark-backups-alt \
+    --config region=us-west-1
+```
+
+During backup creation:
+```shell
+# The Ark server will automatically store backups in the backup storage location named "default" if
+# one is not specified when creating the backup. You can alter which backup storage location is used
+# by default by setting the --default-backup-storage-location flag on the `ark server` command (run
+# by the Ark deployment) to the name of a different backup storage location.
+ark backup create full-cluster-backup
+```
+Or:
+```shell
+ark backup create full-cluster-alternate-location-backup \
+    --storage-location s3-alt-region
+```
+
+#### For volume providers that support it (e.g. Portworx), have some snapshots be stored locally on the cluster and have others be stored in the cloud
+
+During server configuration:
+
+```shell
+ark snapshot-location create portworx-local \
+    --provider portworx \
+    --config type=local
+
+ark snapshot-location create portworx-cloud \
+    --provider portworx \
+    --config type=cloud
+```
+
+During backup creation:
+
+```shell
+# Note that since in this example we have two possible volume snapshot locations for the Portworx 
```

Comment:
Is it worth calling out anywhere that the server also has some additional flags now to indicate which of these  VSL (provider/location tuple) should be the default?